core/src/main/java/io/questdb/griffin/SqlCompiler.java

```diff
@@ -1021,6 +1047,19 @@ private CompiledQuery alterTableRenameColumn(int tableNamePosition, String table
         return compiledQuery.ofAlter(alterOperationBuilder.build());
     }
 
+    private CompiledQuery alterTableResume(int tableNamePosition, String tableName, long resumeFromTxn, SqlExecutionContext executionContext) throws SqlException {
+        try {
+            engine.getTableSequencerAPI().resumeTable(tableName, resumeFromTxn, executionContext.getCairoSecurityContext());
+            return compiledQuery.ofTableResume();
+        } catch (CairoException ex) {
+            LOG.critical().$("table resume failed [table=").$(tableName)
+                    .$(", error=").$(ex.getFlyweightMessage())
+                    .$(", errno=").$(ex.getErrno())
+                    .I$();
+            throw SqlException.position(tableNamePosition).put(ex.getFlyweightMessage()).put("[errno=").put(ex.getErrno()).put(']');
```

Comment:
same thing here, we can probably not re-shape exception

Reply:
Fixed in https://github.com/questdb/questdb/pull/2841/commits/1245bf8516578159fb09b23a263ae9322113cf18